fe1-web/src/features/wallet/screens/__tests__/WalletHome.test.tsx

```diff
@@ -0,0 +1,75 @@
+import { fireEvent, render, waitFor } from '@testing-library/react-native';
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import MockNavigator from '__tests__/components/MockNavigator';
+import { mockLao, mockPopToken, mockRC } from '__tests__/utils';
+
+import { recoverWalletRollCallTokens } from '../../objects';
+import { clearDummyWalletState, createDummyWalletState } from '../../objects/DummyWallet';
+import { RollCallToken } from '../../objects/RollCallToken';
+import { WalletHome } from '../index';
+
+jest.mock('react-redux');
+jest.mock('core/platform/Storage');
+jest.mock('core/platform/crypto/browser');
+jest.mock('features/wallet/objects/Wallet');
+jest.mock('features/wallet/objects/DummyWallet');
+
+beforeEach(() => {
+  jest.clearAllMocks();
+});
+describe('Wallet home', () => {
+  it('renders correctly with an empty wallet', () => {
+    const component = render(<MockNavigator component={WalletHome} />).toJSON();
+    expect(component).toMatchSnapshot();
+  });
+  it('renders correctly with a non empty wallet', async () => {
```

Comment:
for readability you might want to add empty lines between the different `it` clauses

Reply:
I agree!